applications/CompressiblePotentialFlowApplication/python_scripts/apply_far_field_and_wake_process.py

```diff
@@ -3,28 +3,22 @@
 import KratosMultiphysics.CompressiblePotentialFlowApplication as CPFApp
 import math
 
-def DotProduct(A,B):
-    result = 0
-    for i,j in zip(A,B):
-        result += i*j
-    return result
-
 def Factory(settings, Model):
     if( not isinstance(settings,KratosMultiphysics.Parameters) ):
         raise Exception("expected input shall be a Parameters object, encapsulating a json string")
-    return ApplyFarFieldProcess(Model, settings["Parameters"])
+    return ApplyFarFieldAndWakeProcess(Model, settings["Parameters"])
 
 ## All the processes python should be derived from "Process"
-class ApplyFarFieldProcess(KratosMultiphysics.Process):
+class ApplyFarFieldAndWakeProcess(KratosMultiphysics.Process):
     def __init__(self, Model, settings ):
         KratosMultiphysics.Process.__init__(self)
 
         if not settings.Has("angle_of_attack_units"):
-            KratosMultiphysics.Logger.PrintWarning("ApplyFarFieldProcess", "'angle of attack_units' is not provided. Using 'radians' as default angle of attack unit.")
+            KratosMultiphysics.Logger.PrintWarning("ApplyFarFieldAndWakeProcess", "'angle of attack_units' is not provided. Using 'radians' as default angle of attack unit.")
 
         default_parameters = KratosMultiphysics.Parameters( """
             {
-                "model_part_name":"",
+                "far_field_model_part_name":"",
```

Comment:
```suggestion
                "model_part_name":"",
```
As there is only one, keep it like this to be consistent with the rest of Kratos.